FiveCalls/FiveCalls/AppDelegate.swift

```diff
@@ -30,7 +30,12 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         if !UserDefaults.standard.bool(forKey: UserDefaultsKeys.hasShownWelcomeScreen.rawValue) {
             showWelcome()
         }
-        
+
+        var types = UIUserNotificationType()
+        types.insert(.alert)
+        types.insert(.badge)
+        UIApplication.shared.registerUserNotificationSettings(UIUserNotificationSettings(types: types, categories: nil))
```

Comment:
Also, I'd like to move the prompt until the user knows what it is for.

Here's my thought:

- notifications are off by default
- on the welcome screen, just above the 'get started' button, there's a UISwitch that says "Remind me to call", when toggled on, the prompt is displayed. This also pushes the schedule VC.
- if they skip this and just press Get Started, we have another opportunity to present it...
- on the menu -> reminder vc, add a UISwitch in the right nav bar area. When _off_ gray out the entire view (ooh maybe a blur overlay would look nice here).  When _on_, we can prompt for notifications then, and remove the blur to reveal the schedule settings.

(If we don't like the switch on the welcome screen, then maybe a tooltip when they first launch that mentions they can be reminded via the menu...)

Thoughts on this?

(/cc: @mccarron )